medusa/providers/nzb/omgwtfnzbs.py

```diff
@@ -86,7 +86,7 @@ def search(self, search_strings, age=0, ep_obj=None):
 
                 for item in response:
                     try:
-                        if not self._get_title_and_url(item):
+                        if not all(self._get_title_and_url(item)):
```

Comment:
If this really needs to be done to all providers (can't check right now, not even sure what the method exactly does), it should be part of the method.